src/signals/incident/definitions/wizard-step-3-contact.js

```diff
@@ -52,9 +52,9 @@ export default {
       },
       privacy_text: {
         meta: {
-          heading: 'Uw privacy',
+          heading: 'Melding delen',
           value:
-            'We gebruiken uw e-mailadres alléén om u op de hoogte houden, of wanneer wij een vraag hebben en u niet per telefoon kunnen bereiken. We gebruiken uw telefoonnummer alléén om nog iets te kunnen vragen over uw melding.',
+            'Niet alle meldingen vallen onder de verantwoordelijkheid van de gemeenten Amsterdam en Weesp. Om u van dienst te kunnen zijn, kunnen wij uw melding delen met de verantwoordelijke organisaties zoals de Politie of Dierenambulance. Hiervoor hebben we uw toestemming nodig om uw melding te delen.',
```

Comment:
This text will need to be configured in the config file. Since mentioning Weesp and Amsterdam is not going to work for other municipalities.

Reply:
Weesp and Amsterdam are removed from the text now